chat.delta.desktop.yml

```diff
@@ -91,7 +90,8 @@ modules:
       - type: script
         dest-filename: run.sh
         commands:
-          - /app/delta/deltachat-desktop --no-sandbox
+          - export TMPDIR=\"$XDG_RUNTIME_DIR/app/$FLATPAK_ID\"
```

Comment:
You set this yet never grant permission to it.

Reply:
Also why is `/var/tmp` not acceptable, does something from the host use it?

Reply:
AFAIK every app can access `"$XDG_RUNTIME_DIR/app/$FLATPAK_ID"` by default, no?

`/var/tmp` is persistent. There is no reason to keep some trash persistently.

Reply:
Grepping around `flatpak`, nope.

Reply:
Can't find the flatpak code right now but I'm 100% sure that every flatpak app has read-write bind-mount `"$XDG_RUNTIME_DIR/app/$FLATPAK_ID"` from host. Just check yourself.

Reply:
This isn't first time I use it and it works: https://github.com/flathub/org.mozilla.Thunderbird/blob/master/thunderbird-wrapper#L2

Reply:
You are right and this changes everything I ever knew.

Reply:
Also I've added this information to the docs.

Reply:
Which docs?

Reply:
Found it: https://github.com/flatpak/flatpak-docs/commit/19a505c757b98792f4418ce2f2a26910aa4b06b9